### ISSUE TITLE
Jenkins with rollup not working correctly

### ISSUE THREAD

**Issue by semoal** [rollup/rollup#2632]
- Rollup Version: 0.68.1
- Operating System (or Browser): Ubuntu / Jenkins
- Node Version: 11

### How Do We Reproduce?

- Create a rollup file for a react library buliding cjs/esm bundles.
- Create a rollup -c in package.json
- Create a Jenkinsfile, with node and sh to run the command in the package json.
- Deploy, and see Jenkins logs.

### Expected Behavior
Do the same in Jenkins than local 
### Actual Behavior

When I am in local (OS X, High Sierra) and i run npm publish, rollup does all the bundle and finish correctly, but in Jenkins is not waiting to finish and creates the tarball before rollup has finished.
I'm not sure if it's a issue of Jenkins, Rollup, or NPM, but this is so damit strange.

Local publish: ![](https://i.imgur.com/HgoBXTQ.png)
Jenkins publish: 
![captura de pantalla 2019-01-04 a las 8 43 41](https://user-images.githubusercontent.com/22656541/50678012-0e259f80-0ffd-11e9-964f-ec3099482f2e.png)

As you can see Jenkins capture, has the message src/index.js -> ..... but then don't do all of them..

**Comment by semoal** [comment]
I add other image on Jenkins pipeline:
![captura de pantalla 2019-01-04 a las 9 35 57](https://user-images.githubusercontent.com/22656541/50679584-51cfd780-1004-11e9-9f33-1c413cf14264.png)
As you can see, looks like Jenkins can't detect Rollup is doing something, and goes to the next step.

**Comment by semoal** [comment]
Hi, @lukastaegert, thanks for your fast answer.

I've tried almost all the posibilites: 

1. specific script for just building and other for publishing.
2. Custom script with building and publishing, like your performPublish
3. Different stages on Jenkins with individual commands
4. Same stage on Jenkins with all the commands.

And no result... I've posted this here, to try to find a way to fix this, or at least understand why this happens. I've don't expect to get much help but at least listen other points of view.
